test/apps/iodemo/ucx_wrapper.h

```diff
@@ -29,6 +29,8 @@ class UcxConnection;
 struct ucx_request;
 struct UcxAmDesc;
 
+void *UcxAmDesc_get_data(const UcxAmDesc &desc);
+bool UcxAmDesc_is_rndz(const UcxAmDesc &desc);
```

Comment:
`rndv`
also better not to mix Huangrian notation with `_`

Reply:
Does ucx_am_get_data & ucx_am_is_rndv work for you?

Reply:
yes, or `UcxAmDescIsRndv`